examples/tutorial_chef.py

```diff
@@ -0,0 +1,98 @@
+
+    We'll call its `main()` method from the command line script.
+    """
+
+    # 1. PROVIDE CHANNEL INFO  (replace <placeholders> with your own values)
+    ############################################################################
+    channel_info = {    #
+        'CHANNEL_SOURCE_DOMAIN': '<yourdomain.org>',       # who is providing the content (e.g. learningequality.org)
+        'CHANNEL_SOURCE_ID': '<yourid>',                   # channel's unique id
+        'CHANNEL_TITLE': 'The tutorial channel',
+        # 'CHANNEL_THUMBNAIL': 'http://yourdomain.org/img/logo.jpg', # (optional) local path or url to image file
+        # 'CHANNEL_DESCRIPTION': 'What is this channel about?',      # (optional) description of the channel (optional)
+     }
+
+    # 2. CONSTRUCT CHANNEL
+    ############################################################################
+    def construct_channel(self, *args, **kwargs):
+        """
+        This method is reponsible for creating a `ChannelNode` object from the info
+        in `channel_info` and populating it with TopicNode and ContentNode children.
+        """
+        # Create channel
+        ########################################################################
+        channel_info = self.channel_info
+        channel = ChannelNode(
+            source_domain = channel_info['CHANNEL_SOURCE_DOMAIN'],
+            source_id = channel_info['CHANNEL_SOURCE_ID'],
+            title = channel_info['CHANNEL_TITLE'],
+            thumbnail = channel_info.get('CHANNEL_THUMBNAIL'),
+            description = channel_info.get('CHANNEL_DESCRIPTION'),
+        )
+
+        # Create topics to add to your channel
+        ########################################################################
+        # Here we are creating a topic named 'Example Topic'
+        exampletopic = TopicNode(source_id="topic-1", title="Example Topic")
+        # TODO: Create your topic here
+
+        # Now we are adding 'Example Topic' to our channel
+        channel.add_child(exampletopic)
+        # TODO: Add your topic to channel here
+
+        # You can also add subtopics to topics
+        # Here we are creating a subtopic named 'Example Subtopic'
+        examplesubtopic = TopicNode(source_id="topic-1a", title="Example Subtopic")
+        # TODO: Create your subtopic here
+
+        # Now we are adding 'Example Subtopic' to our 'Example Topic'
+        exampletopic.add_child(examplesubtopic)
+        # TODO: Add your subtopic to your topic here
+
+
+        # Content
+        # You can add pdfs, videos, and audio files to your channel
+        ########################################################################
+        # let's create a document file called 'Example PDF'
+        examplepdf = DocumentNode(title="Example PDF", source_id="example-pdf", files=[DocumentFile(path="http://www.pdf995.com/samples/pdf.pdf")], license=licenses.CC_BY_SA)
```

Comment:
Nice! I didn't know about the `get_license` "factory" function...

updating the PR with this

```
fancy_license = get_license(licenses.SPECIAL_PERMISSIONS, description='Special license for ricecooker fans only.', copyright_holder='The chef video makers')
```

produces:

<img width="831" alt="screen shot 2017-07-05 at 9 13 15 pm" src="https://user-images.githubusercontent.com/163966/27891270-dd505250-61c6-11e7-9083-cebb2fb9d52d.png">